### PR TITLE
Added bullet points

### PR DESCRIPTION
**Pull request by bradleysigma**
The list of harassments are now a bulleted list, removing some ambiguity.

### CODE REVIEW

**Comment by mon** [comment]
With bullets, do you still need the semicolons?

**Comment by bradleysigma** [comment]
Because the sentence before the bullets ("Harassment includes, but is not limited to:") ends with a colon, each line in the list should end with a semicolon.

**Comment by mon** [comment]
The typelevel CoC you base this off does not use semicolons

File: README.md [hunk 1]
```diff
@@ -2,7 +2,25 @@
 
 UQCS is dedicated to providing a community for everyone. Community participants violating the following rules may be sanctioned or expelled from the community at the discretion of the executive committee.
 
-We do not tolerate harassment of community participants in any form. Harassment includes offensive comments related to gender or anything to do with it (identity, history, expression, non-binary gender, etc.), sexual orientation, disability, physical appearance, body size, race, religion, programming experience or background, sexual images and languages in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of community activities, spaces, or other events, unwelcome physical contact, and unwelcome sexual attention. Participants asked to stop any harassing behavior are expected to comply immediately.
+We do not tolerate harassment of community participants in any form. Harassment includes, but is not limited to:
+* offensive comments related to:
+  * gender or anything to do with it (identity, history, expression, non-binary gender, etc.);
```
Comment (nicklambourne) [inline README.md]: reword this to:

> gender (including identity, history or expression)

File: README.md [hunk 2]
```diff
@@ -2,7 +2,25 @@
...
+  * sexual orientation;
+  * disability;
+  * physical appearance;
+  * body size;
```
Comment (nicklambourne) [inline README.md]: Can body size be folded into physical appearance?

File: README.md (hunk 1 above)
Comment (bradleysigma) [reply README.md]: That should probably be done in another pull request, given that this pull request is about formatting.

File: README.md [hunk 3]
```diff
@@ -2,7 +2,25 @@
...
+  * programming experience or background;
+* sexual images and languages in public spaces;
+* deliberate intimidation;
+* stalking;
+* harassing photography or recording;
+* sustained disruption of community activities, spaces, or other events;
+* unwelcome physical contact; and
+* unwelcome sexual attention.
+
+Participants asked to stop any harassing behavior are expected to comply immediately.
```
Comment (nicklambourne) [inline README.md]: Potentially follow on with this: 

> Refusal to immediately cease the offending behaviour will result in an escalation - potentially leading to a temporary ban or permanent expulsion.

File: README.md [hunk 4]
```diff
@@ -2,7 +2,25 @@
...
+  * race;
+  * religion; or
+  * programming experience or background;
+* sexual images and languages in public spaces;
+* deliberate intimidation;
+* stalking;
+* harassing photography or recording;
+* sustained disruption of community activities, spaces, or other events;
+* unwelcome physical contact; and
+* unwelcome sexual attention.
```
Comment (nicklambourne) [inline README.md]: Add:

> calls for violence against members 
> incitements to violence or self-harm

File: README.md (hunk 1 above)
Comment (jdcaperon) [reply README.md]: I agree we should move language changes to another PR. @nicklambourne are you happy to resolve your changes?

**Comment by jdcaperon** [comment]
Please remove semicolons for now, this would be in line with most written style guides. 

UQ also has a written style guide that might be worth using as a benchmark if needed. https://omc.uq.edu.au/written-style-guide
